pallets/settlement/src/lib.rs

```diff
@@ -892,6 +899,17 @@ decl_module! {
 
             Self::deposit_event(RawEvent::InstructionRescheduled(did, id));
         }
+
+        /// Edit a venue's signers.
+        /// * `id` specifies the ID of the venue to edit.
+        /// * `signers` specifies the signers to add/remove.
+        /// * `update_type` specifies the update type add/remove of venue.
+        #[weight = <T as Config>::WeightInfo::update_venue_signers(signers.len() as u32)]
+        pub fn update_venue_signers(origin, id: VenueId, signers: Vec<T::AccountId>, update_type: bool) {
```

Comment:
Can you change `update_type` here too?  If `update_type` used an enum, then that name would be ok, but since it is just a `bool`, the current name `update_type` doesn't tell the user what `true` means vs what `false` means.